### DISCUSSION TITLE
mysql_to_mysql operator

### DISCUSSION THREAD

**Discussion by wentao-uw** [apache/airflow#36769] [General]
Write a mysql_to_mysql operator. Here is the link: https://github.com/wentao-uw/airflow-custom-opeartors

**Comment by Taragolis** [comment]
Airflow is primarily a community-driven project, and most of its community members are also active users of the platform, so I would recommend to raise a PR if you would like to add this operator into the [MySQL Provider](https://airflow.apache.org/docs/apache-airflow-providers-mysql/stable/index.html)

Some Useful links which covers contributing to Airflow and Airflow Community Supported Providers:
- [Contributor's Quick Start](https://github.com/apache/airflow/blob/main/CONTRIBUTORS_QUICK_START.rst#contributors-quick-start)
- [CONTRIBUTING.rst](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst)
- [BREEZE.rst](https://github.com/apache/airflow/blob/main/BREEZE.rst)
- [LOCAL_VIRTUALENV.rst](https://github.com/apache/airflow/blob/main/LOCAL_VIRTUALENV.rst)
